functional/basic-attestation-on-localhost/payload-module/local_action_sudo_script.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
This won't work, `payload-module` can only contain Python modules, not bash scripts. You need to rewrite it in a form of a Python module.

Reply:
> You have accidentally added .swp file to this PR.

Thank you for reminding me.

Reply:
Yeah I rewrote it into form of python module.